### PR TITLE
Point release pcmanfm-qt 0.14.1

### PR DESCRIPTION
**Pull request by agaida**

### CODE REVIEW

**Comment by tsujan** [comment]
Good.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,17 @@
+pcmanfm-qt-0.14.0 / 2019-
```
Comment (Chiitoo) [inline CHANGELOG]: Should this be 0.14.1?

Also no date?  (Same for lxqt-config-0.14.1 it seems.)

Comment (agaida) [reply CHANGELOG]: it should - and i wanted to do some tests first before i set the date

Comment (agaida) [reply CHANGELOG]: thanks a lot, damn typos :D

Comment (Chiitoo) [reply CHANGELOG]: Alrighty!  Figured the date was just not set yet, but wasn't sure.  :]

**Pull request merged**